README.rst

```diff
@@ -59,10 +63,8 @@ See the output of ``fitlins --help`` for all valid options::
       -w WORK_DIR, --work-dir WORK_DIR
                             path where intermediate results should be stored
 
-At present, FitLins does not support smoothing or operate in subject-native
-space.
-It is developed against `FMRIPREP`_-preprocessed datasets, but is intended to
-work with any dataset following the `BIDS Derivatives`_ draft specification.
+At present, FitLins does not operate in subject-native space. It is developed against `FMRIPREP`_-preprocessed datasets, 
+but is intended to work with any dataset following the `BIDS Derivatives`_ draft specification.
```

Comment:
Can you just modify the first sentence? I use newlines after each sentence to reduce the diff when sentence length cascades to line changes, as it does here.

Reply:
Thanks! How about now?